### ISSUE TITLE
Settings font size changes not announced by screen reader

### ISSUE THREAD

**Issue by becka11y** [readium/readium-js-viewer#491]
When you first focus the font size slider in the Settings dialog the value is announced by the screen reader (testing with NVDA and JAWS 17, Firefox, windows 7. However, if you press an arrow key to change the value, nothing is announced unless you move focus out of the slider and back to it.  

This is related to #308 but I created a separate issue since this particular item is also a problem with JAWS.

**Comment by danielweck** [comment]
Fixed by https://github.com/readium/readium-js-viewer/pull/492
